code/controllers/subsystem/goldmansachs.dm

```diff
@@ -0,0 +1,137 @@
+SUBSYSTEM_DEF(goldmansachs)
+	name = "Economy"
```

Comment:
Drop the meme and name it SSeconomy

Reply:
Nope, oranges said it could stay as SSgoldmansachs.